bundles/org.openhab.binding.sagercaster/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,214 @@
+                <option value="R2">Precipitation or flurries followed by improvement (within 12 hours)</option>
+                <option value="S">Precipitation or showers/Flurries followed by improvement (within 12 hours) and becoming cooler</option>
+                <option value="S1">Precipitation or showers followed by improvement (within 12 hours) and becoming cooler</option>
+                <option value="S2">Precipitation or flurries followed by improvement (within 12 hours) and becoming cooler</option>
+                <option value="T">Precipitation or showers/Flurries followed by improvement early in period (within 6 hours)</option>
+                <option value="T1">Precipitation or showers followed by improvement early in period (within 6 hours)</option>
+                <option value="T2">Precipitation or flurries followed by improvement early in period (within 6 hours)</option>
+                <option value="U">Precipitation or showers/Flurries by improvement early in period (within 6 hours) and becoming cooler</option>
+                <option value="U1">Precipitation or showers by improvement early in period (within 6 hours) and becoming cooler</option>
+                <option value="U2">Precipitation or flurries by improvement early in period (within 6 hours) and becoming cooler</option>
+                <option value="W">Precipitation or showers/Flurries followed by fair early in period (within 6 hours) and becoming cooler</option>
+                <option value="W1">Precipitation or showers followed by fair early in period (within 6 hours) and becoming cooler</option>
+                <option value="W2">Precipitation or flurries followed by fair early in period (within 6 hours) and becoming cooler</option>
+                <option value="X">Unsettled followed by fair</option>
+                <option value="Y">Unsettled followed by fair early in period (within 6 hours) and becoming cooler</option>
+            </options>
+        </state>
+    </channel-type>
+    
+    <channel-type id="velocity">
+        <item-type>String</item-type>
+        <label>Wind Velocity</label>
+        <state readOnly="true" pattern="%s">
+            <options>
+                <option value="N">Probably increasing</option>
+                <option value="F">Moderate to fresh</option>
+                <option value="S">Strong winds may precede gales over open water)</option>
+                <option value="G">Gale</option>
+                <option value="W">Dangerous gale (whole gale)</option>
+                <option value="H">Hurricane</option>
+                <option value="D">Diminishing, or moderating somewhat if current winds are of fresh to strong velocity</option>
+                <option value="U">No important change. Some tendency for slight increase in winds during day, diminishing in evening</option>
+            </options>
+        </state>
+    </channel-type>
+    
+    <channel-type id="winddirection">
```

Comment:
there seems to be two different naming paradigms here. One uses no spaces between words, the other uses '-'. I would suggest changing them all to the latter.